# HG changeset patch
# Parent 114e9a1d0a73ee418d32dcf74a9d110147df5e57
# Parent  072899cabd49dce8b20e7e799b920ac9b325c06c
Install examples in share/lightsd insted of share/doc/lightsd

Because Debian thinks it's smart to compress files in /usr/share/doc,
which creates a poor user experience for lightsc.py

diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -109,7 +109,7 @@
 )
 INSTALL(
     DIRECTORY examples
-    DESTINATION share/doc/lightsd
+    DESTINATION share/lightsd
     USE_SOURCE_PERMISSIONS
     REGEX ".*\\.sw.$" EXCLUDE
 )
diff --git a/docs/first-steps.rst b/docs/first-steps.rst
--- a/docs/first-steps.rst
+++ b/docs/first-steps.rst
@@ -184,7 +184,7 @@
 
 ::
 
-   `lightsd --prefix`/share/doc/lightsd/examples/toggle
+   `lightsd --prefix`/share/lightsd/examples/toggle
 
 Or, from the root of the repository:
 
@@ -264,7 +264,7 @@
 
 ::
 
-   `lightsd --prefix`/share/doc/lightsd/examples/lightsc.py
+   `lightsd --prefix`/share/lightsd/examples/lightsc.py
 
 Or, from the root of the repository:
 
